keps/sig-apps/2232-suspend-jobs/README.md

```diff
@@ -385,78 +390,86 @@ field.
 
 * **Are there any tests for feature enablement/disablement?** No.
```

Comment:
Please add some unit tests for it.

Reply:
actually, we do have integration tests that test feature enablement exhaustively. updated the KEP.